integrationtests/src/test/java/io/kroxylicious/proxy/multitenant/MultiTenantIT.java

```diff
@@ -265,6 +266,12 @@ private void verifyConsumerGroups(String proxyAddress, String... expectedGroupId
         try (var admin = Admin.create(commonConfig(proxyAddress, Map.of()))) {
             var groups = admin.listConsumerGroups().all().get().stream().map(ConsumerGroupListing::groupId).toList();
             assertThat(groups).containsExactlyInAnyOrderElementsOf(Set.of(expectedGroupIds));
+            var describedGroups = admin.describeConsumerGroups(groups).all().get();
```

Comment:
I'm having second thoughts about the overloading of the test with the responsibility for verify describeGroups.  Maybe a separate test would have been justifiable.  I think I'd like an 'unknowngroup' test case too.

Reply:
I don't disagree with anything you've said, but there's a tension with the 'low bar for a new test':   ITs are slow by their nature (MultiTenantIT are about 6 seconds each).   We need to make sure we favour unit testing wherever possible and have a much smaller set of judiciously chosen ITs that give us the confidence that the system is integrating properly.